### ISSUE TITLE
Make keyboard attribute dynamic

### ISSUE THREAD

**Issue by mmeendez8** [ngx-material-keyboard/core#9]
Hi, I have been trying to make the attribute variable and show the keyboard only under certain conditions but I cannot make it work. This is a simple example of what I have tried using attribute binding:

`[attr.mdKeyboard] = "disabled ? '' : null "`

Any help would be appreciated!!

**Comment by davidenke** [comment]
What you're doing here is setting an HTML attribute. But `mdKeyboard` is an attribute directive (https://angular.io/guide/attribute-directives).

The simplest way would be to use `*ngIf*`: https://stackoverflow.com/a/44631715/1146207
